test/fluree/db/json_ld/api_test.cljc

```diff
@@ -34,3 +35,89 @@
              (is (<p! (fluree/exists? conn ledger-alias)))
              (is (not (<p! (fluree/exists? conn "notaledger"))))
              (done)))))))
+
+#?(:clj
+   (deftest load-from-file-test
+     (testing "can load a file ledger with single cardinality predicates"
+       (with-tmp-dir storage-path
+         (let [conn         @(fluree/connect
+                               {:method :file :storage-path storage-path
+                                :defaults
+                                {:context test-utils/default-context}})
+               ledger-alias "load-from-file-test-single-card"
+               ledger       @(fluree/create conn ledger-alias)
+               db           @(fluree/stage
+                               (fluree/db ledger)
+                               [{:context      {:ex "http://example.org/ns/"}
+                                 :id           :ex/brian,
+                                 :type         :ex/User,
+                                 :schema/name  "Brian"
+                                 :schema/email "brian@example.org"
+                                 :schema/age   50
+                                 :ex/favNums   7}
+
+                                {:context      {:ex "http://example.org/ns/"}
+                                 :id           :ex/cam,
+                                 :type         :ex/User,
+                                 :schema/name  "Cam"
+                                 :schema/email "cam@example.org"
+                                 :schema/age   34
+                                 :ex/favNums   5
+                                 :ex/friend    :ex/brian}])
+               db           @(fluree/commit! ledger db)
+               db           @(fluree/stage
+                               db
+                               ;; test a retraction
+                               {:context   {:ex "http://example.org/ns/"}
+                                :f/retract {:id         :ex/brian
+                                            :ex/favNums 7}})
+               _            @(fluree/commit! ledger db)
+               ;; TODO: Replace this w/ :syncTo equivalent once we have it
+               loaded       (test-utils/retry-load conn ledger-alias 100)]
+           (is (= (:t db) (:t (fluree/db loaded)))))))
```

Comment:
I think I spotted a while ago a latent mismatch between the way `load` calculates the db size and the the way `stage` does, but I haven't had a chance to investigate. Could you humor me and compare the `:stats` of the loaded db to the live db in one of these tests?

Reply:
Yeah, I suspect they won't match - `fluree.db.json-ld.reify/merge-flakes` calculates the size of all the flakes, whereas `fluree.db.json-ld.transact/final-db` calculates the size of the asserts and subtracts the size of the retracts.

I'm not sure which one is correct, but we should be consistent.

Reply:
@dpetran's hunch was correct. These do not match. I have failing test assertions locally, but I don't think pushing them to this PR is in scope. I'll write up a bug card so we can discuss what the consistent approach we want is and then fix it in a separate PR. That seem OK?